multiversx_sdk_core/transaction_builders/delegation_factory.py

```diff
@@ -0,0 +1,430 @@
+from typing import Any, List, Optional, Protocol, Sequence, Tuple
+
+from multiversx_sdk_core import TransactionPayload
+from multiversx_sdk_core.constants import (ARGS_SEPARATOR,
+                                           TRANSACTION_OPTIONS_DEFAULT,
+                                           TRANSACTION_VERSION_DEFAULT)
+from multiversx_sdk_core.errors import ErrListsLengthDoNotMatch
+from multiversx_sdk_core.interfaces import (IAddress, IChainID, IGasLimit,
+                                            IGasPrice, INonce, ISignature,
+                                            ITransactionPayload,
+                                            ITransactionValue,
+                                            IValidatorPublicKey)
+from multiversx_sdk_core.serializer import arg_to_string
+from multiversx_sdk_core.transaction import Transaction
+
+
+class IConfig(Protocol):
+    chain_id: IChainID
+    min_gas_price: IGasPrice
+    min_gas_limit: IGasLimit
+    gas_limit_per_byte: IGasLimit
+
+
+class MetaChainSystemSCsCost:
```

Comment:
These should be merged into `IConfig`. E.g.:

https://github.com/multiversx/mx-sdk-js-core/blob/main/src/tokenOperations/tokenOperationsFactory.ts#L19

Reply:
Merged them into `IConfig`